doc/_templates/sidebar-nav-bs.html

```diff
@@ -1,9 +0,0 @@
-<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
-  <div class="bd-toc-item active">
```

Comment:
Could you replace the class with ``bd-toc-item navbar-nav``

This works locally and keeps the option to configure differently for different pages

Reply:
@phofl - change made, can you double check it's correct? I'm now seeing no dropdowns on the API reference main page:

![image](https://user-images.githubusercontent.com/45562402/190914649-0b0566a8-16b7-4971-b4c8-858182a82dfd.png)

until I click into one

![image](https://user-images.githubusercontent.com/45562402/190914681-9f70313f-01d8-4891-a174-bcb6ce709400.png)

Collapsing it, I only get one dropdown

![image](https://user-images.githubusercontent.com/45562402/190914704-073fe4a2-a59d-4728-8bf3-bc63dce77ebe.png)

I haven't noticed any other changes from removing the file, and this behavior seems worse to me, but it is usable and happy to consider changing after the 1.5 release.

Reply:
I am not sure what the impact of collapse=True/False is, so I am careful here.

The dropdown behaviour is consistent with our current docs (1.4.4 and also 1.3) so this should be ok for now I think?

Reply:
I didn't realize that it is compatible with those previous docs. Sounds good.